MANIFEST

```diff
@@ -0,0 +1,5 @@
+# file GENERATED by distutils, do NOT edit
```

Comment:
not clear exactly what effect these versioning changes are having, but `bam_to_mate_hist.py` is currently broken when i try to run it on my laptop based apparently on version detection problems so hopefully these fix that.

Reply:
I think this will be better than pbr and the ad-hoc approach we had before that. It should be more cross-platform than the shell commands, it doesn't have dependencies like pbr, and if something goes wrong, it should default to version '0-unknown' or something similar. I also checked w/ Shawn on how we're deploying it, and it should be compatible.